environment.yml

```diff
@@ -14,4 +14,6 @@ dependencies:
     - librosa
     - nussl
     - scaper
-    - soxbindings
```

Comment:
Just fyi, this dependency shouldn't be needed as its included in nussl and Scaper.